third_party/qsort_arg_mt.c

```diff
@@ -204,20 +205,42 @@ qsort_arg_mt_internal(void *a, size_t n, intptr_t es,
 		pc -= es;
 	}
 	pn = (char *) a + n * es;
-	r = min(pa - (char *) a, pb - pa);
-	vecswap((char*)a, pb - r, r);
-	r = min(pd - pc, pn - pd - es);
-	vecswap(pb, pn - r, r);
-	if ((r = pb - pa) > es) {
+	d1 = min(pa - (char *) a, pb - pa);
```

Comment:
Actually these four lines of diff are meaningless, the old code (r=;vecswap;r=;vecswap;) was technically the same.

Reply:
Still, I think it's better to apply this diff so that our code doesn't diverge from the origin.